src/pages/GardenAdminView/GardenAdminInfoCard.tsx

```diff
@@ -0,0 +1,63 @@
+import {
```

Comment:
The filename should match the component name it's exporting. Whether that should be GardenAdminInfoCard or GardenInfoCard is up to you.